server/routes.go

```diff
@@ -1470,10 +1504,25 @@ func (s *Server) ChatHandler(c *gin.Context) {
 		defer close(ch)
 		var sb strings.Builder
 		var toolCallIndex int = 0
+		var grammar string
+
+		switch outputFormat, schema := req.GetOutputFormat(); {
```

Comment:
This doesn't seem like the right place for this. The previous location seems more appropriate